efs.tf

```diff
@@ -43,6 +43,18 @@ resource "aws_efs_access_point" "access_point" {
     gid = var.efs_access_point_gid
     uid = var.efs_access_point_uid
   }
+
+  # Since we only use an access point to access the EFS share we need to make sure
+  # the automatically created root directory (on the EFS share) has the correct
+  # permissions or they will override the local configuration.
```

Comment:
```suggestion
  # permissions.
```

Reply:
Do you have any thoughts on tweaking that vs outright removal? I'd like to keep mention that the mount point will override permissions on the local mount point if they don't align.

Reply:
I'm fine if it stays, but I find the "override the local configuration" unclear.  "override the local mount point's permissions" is a better substitute, so I'd be OK with using that instead.

Reply:
@jsf9k Would you look at abb95cd and let me know what you think?

Reply:
Looks good!